lib/bot-service.js

```diff
@@ -44,17 +53,17 @@ botService.post = async (resource, data) => {
                 payload += chunk;
             });
             response.on('end', () => {
-                if (response.statusCode !== 200) {
-                    return reject();
-                }
-                if (payload === '') {
-                    return resolve('{}');
-                }
-                const payloadData = decodeToken(payload);
-                if (payloadData === undefined) {
-                    return reject();
-                }
-                resolve(payloadData);
+              const data = decodeToken(payload);
+              var responseBody = {};
+              if (data instanceof Error)  {
+                reject(data)
```

Comment:
the return reject makes it so you don't have to put an else statement

Reply:
also `;`

Reply:
Done..